### PR TITLE
BF(workaround): mark test_clone.test_autoenabled_remote_msg as known to fail

### PR DESCRIPTION
**Pull request by yarikoptic**
Started to hang, someone needs to troubleshoot.
Sample job
   https://ci.appveyor.com/project/mih/datalad/builds/29552658

So that travis does not bother:
[ci skip]

### CODE REVIEW

File: datalad/distribution/tests/test_clone.py [hunk 1]
```diff
@@ -362,6 +363,9 @@ def test_clone_report_permission_issue(tdir):
         )
 
 
+# Started to hang on appveyor, only in the first run
```
Comment (kyleam) [inline datalad/distribution/tests/test_clone.py]: As I just saw you note in another PR, this isn't specific to a particular run.

Comment (kyleam) [reply datalad/distribution/tests/test_clone.py]: > As I just saw you note in another PR

Never mind, [that][0] is about the codecov failure, so please ignore this bit of my statement.

[0]: https://github.com/datalad/datalad/pull/3931#issuecomment-566147454

Comment (kyleam) [reply datalad/distribution/tests/test_clone.py]: Hmm, I see you marked this as resolved, but you didn't say how it was (and I don't see an additional push).  Unless I'm misreading your comment, I provided several examples that showed it was inaccurate.

Comment (yarikoptic) [reply datalad/distribution/tests/test_clone.py]: I thought 'nevermind' was the resolution

Comment (kyleam) [reply datalad/distribution/tests/test_clone.py]: I quoted a specific piece of my previous commented, and said "ignore this bit of my statement".

Comment (yarikoptic) [reply datalad/distribution/tests/test_clone.py]: ok, then I misunderstood you.  Please to push a fix to the comment

Comment (kyleam) [reply datalad/distribution/tests/test_clone.py]: Done as part of gh-3949.

**Pull request merged**